src/components/main/Main.js

```diff
@@ -1,18 +1,61 @@
 import TodoList from '../todoList';
-import UserInfo from '../userInfo';
-
-const Main = () => (
-  <main>
-    <h1>React Todo</h1>
-    <TodoList />
-    <UserInfo
-      name="Vladislav"
-      categories={10}
-      published={3}
-      products={50}
-      link="#"
-    />
-  </main>
-);
+import {Tabs, Tab} from '../tabs';
+import Login from '../../pages/login';
+import {todoList} from '../../services/todoList';
+import {checkUser} from '../../services/user.js';
+
+class Main extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      list: null,
+      user: null
+    };
+
+    this.days = ['ПН', 'ВТ', 'СР', 'ЧТ', 'ПТ', 'СБ', 'ВС'];
+    this.currentDay = new Date().getDay() - 1;
+  }
+
+  componentDidMount() {
+    todoList()
+      .then(list => this.setState({ list }));
+    checkUser()
+      .then(user => this.setState({ user }))
+  }
+
+  onLogin = (user) => {
+    this.setState({ user });
+  };
+
+  render() {
+    const { list, user } = this.state;
+
+    return (
+      <main>
+        <h1>React Todo</h1>
+        {
+          user
+            ? <h3>Hello !</h3>
+            : <Login onLogin={this.onLogin}/>
+        }
+        {
+          list
+            ? <Tabs selectedIndex={this.currentDay}>
+              {
+                list.map((elem, index) => (
+                  <Tab title={this.days[index]} key={index}>
+                    <TodoList list={elem}/>
+                    <a href="/">Добавить новый</a>
+                  </Tab>
+                ))
+              }
+              </Tabs>
+            : 'Loading...'
```

Comment:
I see `loading...` before any actions.
http://i.imgur.com/rQYnKXD.png

And after I logged in I still see `loading...`
http://i.imgur.com/K52eFsK.png